app/views/guests/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1> <%= @guest.name %>'s Page</h1>
+<br>
+<h4>Past Stays:</h4>
+<ul>
+  <% @guest.rooms.each do |room| %>
+    <li> <%= room.suite %> - rate: $<%= room.rate %>/night, hotel: <%= room.hotel.name %></li>
+    <br>
+  <% end %>
+</ul>
+  
+<%= form_with url: "/guests/#{@guest.id}/guest_rooms", method: :post, local: true do |form|  %>
```

Comment:
I think the form body is good and the method is also good, but I think the URL of this form could be more RESTful. 

When we're writing paths, consider the format of "resource/:which_one". So if we're POST-ing something, that means we're creating something. I like to ask myself the following kinds of questions: 
- What thing are we creating here? We're creating a relationship between a Guest and a Room. 
- Do we have a model for that? Yes, it's `GuestRoom`. So, our resource (URL) would be `"guest_rooms"`. 

Since we're creating this relationship, both of the IDs we need to do it could be contained within the request body, the form data. Since the user is only rqeuired to enter the Room Id, we can make a hidden field for `guest_id` and fill in that value with `@guest.id`.

Another potential path you could use would be
`url: "/guests/:guest_id/rooms`
and this would still be a POST to the `guest_rooms_controller#create` action.

Reply:
To be clear, the reason I would just do guest_rooms for the URL is because that's the resource we're creating.
The alternative of doing `guests/:guest_id/rooms` is because we're adding a specific room to this specific guest.

I've seen and used both types of paths for this kind of operation and I think both are equally acceptable.